classes/blocks/class-socialmediacards.php

```diff
@@ -30,22 +30,34 @@ public function __construct() {
 				'editor_script'   => 'planet4-blocks',
 				'render_callback' => [ $this, 'render' ],
 				'attributes'      => [
-					'id'                         => [
-						'type' => 'integer',
-					],
-					'title'                      => [
+					'title'       => [
 						'type'    => 'string',
 						'default' => '',
 					],
-					'description'                => [
+					'description' => [
 						'type'    => 'string',
 						'default' => '',
 					],
-					'gallery_block_focus_points' => [
-						'type' => 'string',
-					],
-					'multiple_image'             => [
-						'type' => 'string',
+					'cards'       => [
+						'type'    => 'array',
+						'default' => [],
+						'items'   => [
+							'type'       => 'object',
+							'properties' => [
+								'image_id'     => [
+									'type' => 'int',
+								],
+								'message'      => [
+									'type' => 'string',
+								],
+								'social_url'   => [
+									'type' => 'string',
+								],
+								'focal_points' => [
```

Comment:
Looks like it left back...

Reply:
I kept it here intentionally so that the backend already regards blocks having that attribute as valid, since we will probably be adding it later.

If it is not specified here and there is a block that has a single card with the attribute, the behavior of wordpress is pretty weird, in that it will not fail but just discard the entire "cards" attribute but preserve the rest of the block.

So if we would at a later point add the focal points but have to revert the code to this state for some reason, then the cards with the `focal_points` attribute will still have a "valid" definition and keep on displaying.